### DISCUSSION TITLE
有没有可能支持自定义模板

### DISCUSSION THREAD

**Discussion by Elite-stay** [xmake-io/xmake#5731] [Ideas]
就是create的时候选的工程模板，有时候有些工程配置有一部分是重复的，比如固定添加的包、选项等等，如果可以自定义模板的话，就不用每次新建完工程之后复制粘贴了

**Comment by waruqi** [comment]
https://github.com/xmake-io/xmake/issues/3539

**Reply by Elite-stay** [reply]
> #3539

好像要考虑的确实挺多的，用户只管写`{{{targetname}}}`就行了
